frontend/actions/src/main/kotlin/com/virtuslab/gitmachete/frontend/actions/dialogs/DivergedFromRemoteDialog.kt

```diff
@@ -0,0 +1,104 @@
+package com.virtuslab.gitmachete.frontend.actions.dialogs
+
+import com.intellij.openapi.project.Project
+import com.intellij.openapi.ui.DialogWrapper
+import com.intellij.ui.dsl.builder.MutableProperty
+import com.intellij.ui.dsl.builder.panel
+import com.virtuslab.gitmachete.backend.api.IManagedBranchSnapshot
+import com.virtuslab.gitmachete.backend.api.IRemoteTrackingBranchReference
+import com.virtuslab.gitmachete.backend.api.SyncToRemoteStatus
+import com.virtuslab.gitmachete.frontend.actions.compat.buttonsGroupCompat
+import com.virtuslab.gitmachete.frontend.actions.compat.commentCompat
+import com.virtuslab.gitmachete.frontend.actions.compat.rowCompat
+import com.virtuslab.gitmachete.frontend.resourcebundles.GitMacheteBundle.fmt
+import com.virtuslab.gitmachete.frontend.resourcebundles.GitMacheteBundle.getString
+
+enum class DivergeResolutionOption {
```

Comment:
Nit:
```suggestion
enum class DivergenceResolutionOption {
```
here and in the field